src/components/MyKiva/BadgeModal.vue

```diff
@@ -1,5 +1,21 @@
 <template>
-	<KvLightbox :visible="show" :title="title" @lightbox-closed="closeLightbox">
+	<KvLightbox
+		:class="{'badge-modal': !isJourneyActive}"
+		:visible="show"
+		:title="title"
+		@lightbox-closed="closeLightbox"
+	>
+		<template v-if="!isJourneyActive" #header>
```

Comment:
I think we also need to check `isEarnedSection` somewhere in this new template code. The reason being is that the modal should close and not return to the journey if a completed badge is clicked in the bottom section of MyKiva, where the completed badges are shown.